src/plugins/wb-delete-pii/wb-delete-pii.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * @title WET-BOEW wb-delete-pii
+ * @overview This plugin delete Personal Identifiable Information (PII) before form submit
+ * @license wet-boew.github.io/wet-boew/License-en.html / wet-boew.github.io/wet-boew/Licence-fr.html
+ * @author @polmih
+ **/
+( function( $, wb ) {
+"use strict";
+
+var $document = wb.doc,
+	componentName = "wb-delete-pii",
+	selector = "." + componentName,
```

Comment:
Should the selector be the data attribute or only the css selector.

Reply:
the data attribute. Thanks